docs/api.md

```diff
@@ -44,44 +43,41 @@ import { mount } from '../test-utils/enzyme'; // import from the module defined
 import MyComponent from './MyComponent';
 
 describe('MyComponent', () => {
-  let component;
-  let store;
-  let history;
+  let wrapper;
```

Comment:
I decided to rename all the `component`s to `wrapper` across all the docs to be more like enzyme.